tests/virt_autotest/transactional/prepare_transactional_server.pm

```diff
@@ -82,7 +88,7 @@ sub prepare_packages {
         save_screenshot;
 
         my $cmd = "--non-interactive install --no-allow-downgrade --no-allow-name-change --no-allow-vendor-change";
-        $cmd = $cmd . " $_" foreach (split(/,/, get_required_var("INSTALL_OTHER_PACKAGES")));
+        $cmd = $cmd . " $_" foreach (split(/,/, $additional_pkgs));
```

Comment:
Why this change ?

Reply:
The same reason with the comment https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15878#discussion_r1028045110